server/routes/find-location.js

```diff
@@ -17,11 +17,11 @@ module.exports = [{
     } else {
       // Error
       const err = request.yar.get('displayError')
-      request.yar.set('displayError', {})
-      const location = request.yar.get('locationError').input
-      request.yar.set('locationError', {})
+      request.yar.set('displayError', null)
```

Comment:
setting the yar var to {} meant the bool test was not returning falsey when this was set, so needs setting to null

Reply:
In fact removing locationError has it is unnecessary when we have the displayError for the purpose of holding the user input